src/etc/inc/util.inc

```diff
@@ -1142,6 +1147,11 @@ function is_portoralias($port) {
 	}
 }
 
+/* returns true if $port is a valid TCP/UDP port number or range ("<port>:<port>") or an alias thereof */
+function is_portorrangeoralias($port) {
+	return (is_portoralias($port) || is_portrange($port));
```

Comment:
@rbgarga Before I refactor more stuff here, and have more commits that may or may not be what you are thinking of, please confirm what you would like.
1) I could put the "_" in function names is_port_or_alias is_port_or_range is_port_or_range_or_alias ... and change all uses of those throughout the system.
2) I could refactor the existing is_portoralias() to split out the code block that effectively does is_portalias() and then make is_portoralias() be something like:
```
    return (is_port || is_port_alias);
```
What takes your fancy?